src/test/java/com/epam/test_generator/dao/interfaces/SuitDAOTest.java

```diff
@@ -196,9 +190,35 @@ public void testUpdateName() {
         Assert.assertEquals(newSuit, suitDAO.getOne(id));
     }
 
+    @Test
+    public void testUpdateRowNumber() {
+
+        suitDAO.save(Arrays.asList(
+                new Suit(null, "1", "1", new ArrayList<>(), 1, new HashSet<>(), 1),
+                new Suit(null, "2", "2", new ArrayList<>(), 2, new HashSet<>(), 2),
+                new Suit(null, "3", "3", new ArrayList<>(), 3, new HashSet<>(), 3)
+        ));
+
+        final Map<Long, Integer> patch = new HashMap<>();
```

Comment:
Why patch is in this test case at all? It makes test case logic complicated and takes attention out of piece of logic being tested.

Suit suit1 = suitDAO.findOne(1);
suit1.setRowNumber(2);
Suit suit2 = suitDAO.findOne(2);
suit2.setRowNumber(6);
Suit suit3 = suitDAO.findOne(3);
suit3.setRowNumber(5);

suitDAO.save(Arrays.asList(suit1, suit2, suit3));
assertEquals(2, suitDAO.findOne(1).getRowNumber());
assertEquals(6, suitDAO.findOne(2).getRowNumber());
assertEquals(5, suitDAO.findOne(3).getRowNumber());

Don't make test case too complicated for such simple scenario.